app/default/routes.py

```diff
@@ -23,3 +24,15 @@ def not_found(error):
 @default_bp.errorhandler(500)
 def internal_server_error(error):
     return render_template("500.html"), 500
+
+
+@default_bp.route("/help", methods=["GET"])
```

Comment:
Could we use `@login_requested` decorator instead i.e. https://github.com/communitiesuk/funding-service-design-utils/blob/bafc22f12d359ccad3dc62ee8830d68bba279e8c/fsd_utils/authentication/decorators.py#L115 which makes auth optional rather than having this in unprotected routes as present?

Reply:
From what I can tell `@login_requested` is already called before each request:

https://github.com/communitiesuk/funding-service-design-assessment/blob/6ab022e83cdbe1da40861026dcd84e3058fbc09b/app/create_app.py#L119-L125

So adding `"/help"` was essentially making it an unprotected route, with the change you're suggesting, I don't think we need the `@login_requested` decorator as the `@flask_app.before_request` handles it - I could be misunderstanding something though so let me know if you think I'm missing something.

In short, do we want the `"/help"` route to be protected or not? If not, this PR has that, if we do - I can remove `"help"` from the `unprotected_routes` variable 👍

Reply:
Yeah fair point I'll double-check.

Reply:
Spoke with Emma again yesterday and decided to make it protected for now, have removed "help" from the unprotected routes in https://github.com/communitiesuk/funding-service-design-assessment/pull/175/commits/909df509571a65bf731b81774764184511c83b6f

Reply:
Spoke with Emma again yesterday and decided to make it protected for now, have removed "help" from the unprotected routes in https://github.com/communitiesuk/funding-service-design-assessment/pull/175/commits/909df509571a65bf731b81774764184511c83b6f